src/components/AppNavHeader.tsx

```diff
@@ -15,21 +16,35 @@ export function AppNavHeader({
   const title = options.title || route.name;
 
   return (
-    <Appbar.Header style={styles.style}>
+    <Appbar.Header style={styles.style} elevated>
       {back ? (
         <Appbar.BackAction
           onPress={navigation.goBack}
           style={styles.backAction}
         />
       ) : null}
-      <Appbar.Content title={title} style={styles.content} />
+      <Appbar.Content
+        title={<Title text={title} style={styles.titleText} />}
+        style={styles.content}
+      />
     </Appbar.Header>
   );
 }
 
-const defaultStyles = createStyles('AppNavHeader', () => ({
-  style: {},
+const Title = ({ text, style }: { text: string; style?: TextStyle }) => (
+  <Text variant="titleMedium" style={style}>
+    {text}
+  </Text>
+);
+
+const defaultStyles = createStyles('AppNavHeader', (theme) => ({
+  style: {
+    backgroundColor: '#fff',
```

Comment:
This should maybe come from the theme?

Reply:
Yea it should.  However, the theme doesn't have a full white or full black specifically because each token needs to map to a different lightness (tone) of the same color depending on if it is in light or dark mode.  The closest it has is `background` which maps to `neutral99` in light mode - one step lightness away from pure white.  But I haven't set up translucent headers yet (another PR) and so you can see that band of slightly tinted white vs pure white in the notch area.

This will be changed to `theme.colors.background` in that PR.